app/javascript/components/forms/Modal.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+  <div
+    v-if="showModal"
+    class="modal"
+  >
+    <div class="modal__inner">
+      <div class="modal__body">
+        <div class="modal__content">
+          <p class="modal__question">
+            <b v-text="modalText.question" />
+          </p>
+          <div
+            class="modal__question-body"
+            v-html="modalQuestionBody"
+          />
+          <button
+            class="modal__button modal__button--continue"
+            v-text="modalText.continue"
+            @click="closeModal"
+          />
+          <button
+            class="modal__button modal__button--exit"
+            v-text="modalText.exit"
+            @click="exit"
```

Comment:
could we make this button functional? So maybe add a link to locales and pass it here.

Reply:
Didn't add to locales as it's either going to call `survey.completeLastPage()` or redirect to `/dashboard` (depending on the form), but I've added that functionality. It currently works on the criteria form, but we need to config hash setting up for it to work on the commitments form